powerauth-webflow-authentication-init/src/main/java/io/getlime/security/powerauth/lib/webflow/authentication/method/init/controller/ApiController.java

```diff
@@ -58,8 +71,12 @@
         if (operation == null) {
             final String operationName = "login";
             final String operationData = "{}";
+            AbstractMessageSource messageSource = i18nService.getMessageSource();
+            final OperationFormData formData = new OperationFormData();
+            formData.addTitle("operation.title", "login.title");
+            formData.addMessage("operation.message", "login.message");
```

Comment:
In current implementation you can use the following code:
formData.addTitle("login.title");
formData.addMessage("login.message");

This way you set the ID of i18n key for different title and message depending on the operation. So if you want to initialize payment instead of login operation, you can use:
formData.addTitle("payment.title");
formData.addMessage("payment.message");

And so on.

As long as the i18n keys are present, this will work, because Web Flow will localize the value as well as translate it using the {xyz} patterns.

I realize that this implementation is not ideal, because we want to separate ID for i18n from ID of the form field to make title and message work a bit more like web input fields. However currently the JS frontend supports only one title and one message (and mToken app as well), so the IDs of these form fields are quite irrelevant. I suggest to postpone extending OperationFormData to future release.

I raised issue to extend OperationFormData and separate ID of field from ID for i18n:
https://github.com/lime-company/powerauth-webflow/issues/221